starter-code/src/merge-sort.js

```diff
@@ -1,27 +1,32 @@
 function mergeSort(array) {
-  if(array.length < 2) { return array }
+  if(array.length < 2) {
+    return array;
+  }
 
-  var middle  = Math.floor(array.length / 2);
-  var left    = array.slice(0, middle);
-  var right   = array.slice(middle);
+  const middle  = Math.floor(array.length / 2);
+  const left    = array.slice(0, middle);
+  const right   = array.slice(middle);
 
   return sortHalves(mergeSort(left), mergeSort(right));
 }
 
 function sortHalves(left, right) {
-  var array = [];
+  let array = [];
 
   while(left.length && right.length) {
```

Comment:
What is this loop for? Add more comments to clarify why you did something, and what it's use is